lib/shore/client/tokens/access_token.rb

```diff
@@ -9,29 +8,21 @@ module Tokens
       # @example JWT Payload
       # {
       #   "exp": 1455120875,
+      #   "id": "226fc766-3cf0-4d18-a988-5f8235f17edb",
+      #   "type": "merchant-account",
       #   "data": {
```

Comment:
"data". The idea here is that access tokens are a wrapper for any data. So type might be "something-completely-different" and the corresponding data & class which reads the data will be different.

For now "merchant-account" is parsed by `V1::MerchantAccount`